mutiny-server/src/extractor.rs

```diff
@@ -0,0 +1,33 @@
+use async_trait::async_trait;
+use axum::extract::{rejection::FormRejection, Request, FromRequest};
+use axum::http::StatusCode;
+use axum::Json;
+use serde_json::{json, Value};
+
+pub struct Form<T>(pub T);
```

Comment:
why forms? is that what phoenixd is using?

Reply:
yeah, that's what they are doing